pkg/hive/manager.go

```diff
@@ -263,3 +266,19 @@ func (hr *clusterManager) installLogsForLatestDeployment(ctx context.Context, cd
 
 	return latestProvision.Spec.InstallLog, nil
 }
+
+func (hr *clusterManager) GetSyncSetResources(ctx context.Context, doc *api.OpenShiftClusterDocument) (*hivev1alpha1.ClusterSync, error) {
+	clusterSync := &hivev1alpha1.ClusterSync{}
+
+	key := client.ObjectKey{
+		Name:      ClusterDeploymentName, // "cluster",
+		Namespace: doc.OpenShiftCluster.Properties.HiveProfile.Namespace,
+	}
+
+	err := hr.hiveClientset.Get(ctx, key, clusterSync)
+	if err != nil {
+		log.Fatalf("Error getting ClusterSync resources: %s", err.Error())
```

Comment:
Should it exit when it failed?
https://pkg.go.dev/github.com/sirupsen/logrus#Fatalf

```suggestion
		log.Errorf("Error getting ClusterSync resources: %s", err.Error())
```

Reply:
updated